WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacyBannerViewController.swift

```diff
@@ -7,6 +7,12 @@ import WordPressUI
 ///
 final class PrivacyBannerViewController: UIHostingController<PrivacyBanner> {
 
+    // These are part of the `DrawerPresentable` protocol.
+    // We define them here instead than in the existing extension that defines the protocol conformance so that we can make the stored constants.
+    // Otherwise, since extensions don't allow for stored properties, they would have had to be computed vars.
+    let allowsTapToDismiss = false
+    let allowsDragToDismiss = false
```

Comment:
I think we will also need `let allowsUserTransition = false` to prevent all gestures from dismissing the app.

[Q]: What problem do you see in making these computed vars? I think it may be easier for future maintainers to find all of these properties under the same protocol extension, what do you think? 😅

Reply:
Discoverability is a legit issue 🤔 

> What problem do you see in making these computed vars?

These values don't change, it seems inefficient to compute them at runtime when they could be set once.

Having said that, if it helps with discoverability, I'm happy to move them in the extension.

Reply:
Addressed in 5fc82070fa2dce7abddc8181959df3ade454de42